scripts/ci/run_perftest.sh

```diff
@@ -258,92 +177,158 @@ curl -XPOST -H "Authorization: Token ${TEST_TOKEN}" \
   http://${NGINX_HOST}:8086/api/v2/dbrps
 }
 
-##########################
-## Run and record tests ##
-##########################
-
-# Generate and ingest bulk data. Record the time spent as an ingest test if
-# specified, and run the query performance tests for each dataset.
-for usecase in iot metaquery multi-measurement; do
-  USECASE_DIR="${DATASET_DIR}/$usecase"
-  mkdir "$USECASE_DIR"
-  data_fname="influx-bulk-records-usecase-$usecase"
+bulk_data_file_loader() {
+  local data_fname="influx-bulk-records-usecase-$test_name"
+  # Note -scale-var=1000 implies cardinality of 1MM
```

Comment:
I believe this is only true for the metaquery data generator. Other data generators like the IoT generator use it differently.

Reply:
Ah - I was preserving the previous comment. Will remove it.